test.sh

```diff
@@ -1,8 +1,9 @@
 #!/usr/bin/env bash
 
 function finish {
+  echo '-----------------------test.sh------------------------------'
   echo 'Removing test environment'
-  echo '-----'
+  echo '------------------------------------------------------------'
```

Comment:
Why are these lines so long? `-----` is good enough i think

Reply:
From my opinion it is making the log much more clear